### PR TITLE
Add iRT support for CLI import peptide search

### PR DESCRIPTION
**Pull request by kaipot**

### CODE REVIEW

File: pwiz_tools/Skyline/CommandLine.cs [hunk 1]
```diff
@@ -1731,10 +1731,21 @@ private bool ImportSearch(CommandArgs commandArgs)
         private bool ImportSearchInternal(CommandArgs commandArgs, ref SrmDocument doc)
         {
             var progressMonitor = new CommandProgressMonitor(_out, new ProgressStatus(String.Empty));
+            IrtStandard irtStandard = null;
+            if (!string.IsNullOrEmpty(commandArgs.IrtStandardName))
+            {
+                irtStandard = Settings.Default.IrtStandardList.FirstOrDefault(standard => Equals(standard.Name, commandArgs.IrtStandardName));
```
Comment (nickshulman) [inline pwiz_tools/Skyline/CommandLine.cs:1737]: IrtStandardList is a XmlMappedList, so I believe you can do:
irtStandard = Settings.Default.IrtStandardList[commandArgs.IrtStandardName];

**Review by nickshulman — APPROVED** [review]
Looks good to me.

File: pwiz_tools/Skyline/CommandLine.cs (hunk 1 above)
Comment (kaipot) [reply pwiz_tools/Skyline/CommandLine.cs:1737]: It looks like that will throw an exception if it isn't in the list, though

Comment (nickshulman) [reply pwiz_tools/Skyline/CommandLine.cs:1737]: Oh, right, good point.

**Pull request merged**